sdk/documenttranslator/ai-document-translator-rest/samples/v1/javascript/README.md

```diff
@@ -0,0 +1,66 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-cognitive-services
+  - azure-document-translator
+urlFragment: ai-document-translator-javascript
+---
+
+# Azure Document Translator client library samples for JavaScript
```

Comment:
We'll probably need some way to call out that these samples are specifically for the low level REST client.

Reply:
I can a use "productName": "Azure Document Translator rest", in the sample metadata